Automattic/wp-calypso
# Stats: Fixing domain banner position

Related to #76952

## Proposed Changes

* Moving domain banner tip above the navigation for the `Insights` and `Subscribers` pages
* Wrapping the component in a `div` to capture CSS that adds padding to all page elements

## Testing Instructions



* open the live branch
* go to a blog that doesn't have a custom domain assigned
* go to the `Insights` page and verify that the domain banner has left and right padding and is visible at the top of the page like the screenshot below

![SCR-20230607-lfup](https://github.com/Automattic/wp-calypso/assets/112354940/6bcb673b-28df-4c2c-aede-5d0553db07c6)


## Pre-merge Checklist
